src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/latest/test_storage_container_legal_hold.py

```diff
@@ -0,0 +1,29 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import (ScenarioTest, JMESPathCheck, ResourceGroupPreparer, StorageAccountPreparer)
+
+
+class StorageLegalHold(ScenarioTest):
+    @ResourceGroupPreparer()
+    def test_legal_hold(self, resource_group):
+        storage_account = self.create_random_name('clistorage', 20)
+        self.cmd('storage account create -g {} -n {} --kind StorageV2'.format(
+            resource_group, storage_account))
+        container_name = 'container1'
+        self.cmd('storage container create --account-name {} -n {}'.format(storage_account, container_name))
+
+        self.cmd('storage container legal-hold show --account-name {} -c {} -g {}'.format(
+            storage_account, container_name, resource_group), checks=[
+                JMESPathCheck("tags", [])])
```

Comment:
Hey willie, does this check that the tags field returned is an empty list?

Reply:
Yes, it does.